tests/src/sanity.rs

```diff
@@ -531,8 +531,12 @@ mod tests {
         let mut process = application
             .run(&service.pod_name, Some(&service.namespace), agent.flag())
             .await;
-        process.wait_for_line(Duration::from_secs(30), "daemon subscribed");
+        process.wait_for_line(Duration::from_secs(120), "daemon subscribed");
```

Comment:
How come we have to wait 120 seconds(max) for this message?

Reply:
It's just to reduce flakiness - the macOS machines are a bit slow and we run the tests concurrently so sometimes launching the agent and connecting to it takes time, so..